app/api/dao/user.py

```diff
@@ -188,3 +190,79 @@ def authenticate(username_or_email, password):
             return user
 
         return None
+
+    @staticmethod
+    def get_achievements(user_id):
+        """
```

Comment:
In front of `"""` you can have a small summary of the function

You can add this:
```
"""Shows a subset of the user's achievements
```